### DISCUSSION TITLE
Add new metadata fields

### DISCUSSION THREAD

**Discussion by Iwuaa** [SilentVoid13/Templater#1426] [Help]
If my note already has metadata before inserting template that also has metadata how do I ensure they are added to the existing ones and nothing breaks? 
Seems like a very basic question but I couldn't find an answer.

What I want to achieve:
**MyFile.md**
```
---
property1 :  value1
property2 :  value2
---
File content
```

**MyTemplate.md** 
```
---
property3 : value3
property4 : value4
---
```

**Actual result**
```
---
property1 :  value1
property2 :  value2
---
---
property3 : value3
property4 : value4
---
File content
```

**Expected result**
```
---
property1 :  value1
property2 :  value2
property3 : value3
property4 : value4
---
File content
```

**Comment by Zachatoo** [comment]
There's a feature request for this here #1218, I would prefer to keep discussion around this there
